packages/graphql-transformers-e2e-tests/src/__tests__/FunctionTransformerTestsV2.e2e.test.ts

```diff
@@ -39,9 +46,98 @@ function outputValueSelector(key: string) {
     const output = outputs.find((o: Output) => o.OutputKey === key);
     return output ? output.OutputValue : null;
   };
-}
+};
+
+const createEchoFunctionInOtherAccount = async (currentAccountId?: string) => {
+  if (!currentAccountId) {
+    return;
+  }
+  try {
+    const childAccounts = (await organizations.listAccounts({}).promise())?.Accounts;
+    if (!childAccounts || childAccounts?.length < 1) {
+      console.warn('Could not find any child accounts attached to current account');
+      expect(true).toEqual(false);
+      return;
+    }
+    const otherAccountId = childAccounts[0]?.Id;
+    if (!otherAccountId) {
+      console.warn('Could not choose other account to create lambda function');
+      expect(true).toEqual(false);
+      return;
+    }
+    const childAccountRoleARN = `arn:aws:iam::${otherAccountId}:role/OrganizationAccountAccessRole`;
+    const accountCredentials = (await sts.assumeRole({ 
+      RoleArn: childAccountRoleARN, 
+      RoleSessionName: `testCrossAccountFunction${BUILD_TIMESTAMP}`,
+      DurationSeconds: 900
+    }).promise())?.Credentials;
+    if (!accountCredentials?.AccessKeyId || !accountCredentials?.SecretAccessKey || !accountCredentials!.SessionToken) {
```

Comment:
Is the second `!` in `!accountCredentials!.SessionToken` intentional?

Reply:
Definitely not, changed it.